docs/CLI.md

```diff
@@ -96,8 +108,12 @@ DESCRIPTION:
    and symbol information.
 
 OPTIONS:
-   --from value  Path to SCIP index file (default: index.scip)
-   --to value    Path to output directory for snapshot files (default: scip-snapshot)
+   --comment-syntax value  Comment syntax to use for snapshot files (default: "//")
+   --from value            Path to SCIP index file (default: "index.scip")
+   --help, -h              show help (default: false)
+   --project-root value    Override project root in the SCIP file. For example, this can be helpful when the SCIP index was created inside a Docker image or created on another computer
+   --strict                If true, fail fast on errors (default: true)
+   --to value              Path to output directory for snapshot files (default: "scip-snapshot")
```

Comment:
Q: Why is this PR changing these lines? I thought the earlier PR that verified the documentation should've fixed this... 🤔

Reply:
The tests only verify that the global CLI help text is valid, not for any of the subcommands :(

It won't be much of an issue to add extra checks in a separate PR.

Reply:
Could you remove the unrelated changes from this PR? I can submit a PR tomorrow making the test more robust and fix this.

Reply:
> Could you remove the unrelated changes from this PR?

IMO, this is not a codebase with ridiculous velocity where unrelated changes are a problem - ongoing small improvements that aren't at odds with main purpose of the PR shouldn't be penalised by extra effort of reverting and resubmitting. 

There are many small things that can be improved for which there is no plan and no dedicated effort - with a small number of contributors we should aim to fix things as we go.

You are free to follow up with a test for this (or I can do this) but updating and fixing small things in the docs should be a welcome addition to any PR.

Reply:
First of all, my comment was not a blocking one.

> ongoing small improvements that aren't at odds with main purpose of the PR shouldn't be penalised by extra effort of reverting and resubmitting.

I'm not trying to "penalise" something here... Ideally, these would not be part of the same PR (or commit) anyways. Normal commit hygiene IMO dictates that you'd commit related stuff in a single commit and move unrelated commit into separate PRs before submitting PRs. Even it got bundled into the same PR, if it were in different commits, it'd be easy to revert and push. 

That's regardless of codebase velocity etc. My point isn't about velocity, it's about completeness. With drive-by improvements that get mixed in with other PRs, it's less clear what still needs fixing vs what doesn't. E.g. here, if I hadn't flagged this, then the point about the test missing might not have been noticed.

That said, I don't think it's a big deal to leave it in.